libraries/DataFlash/LogStructure.h

```diff
@@ -91,6 +91,7 @@ const struct UnitStructure log_Units[] = {
     { 'G', "Gauss" },         // Gauss is not an SI unit, but 1 tesla = 10000 gauss so a simple replacement is not possible here
     { 'h', "degheading" },    // 0.? to 359.?
     { 'i', "A.s" },           // Ampere second
+    { 'I', "Instance" },      // Message contains instance fields which should be graphed seperately
```

Comment:
Comment needs help.  I don't know what, though.

Reply:
Agreed, I was unsure where to go with it and it shows :)